packs/vsphere/actions/vm_env_items_get.py

```diff
@@ -50,12 +50,12 @@ def run(self, itemtype, parents=False, summary=False, vsphere=None):
         itemlist = inventory.get_managed_entities(self.si_content, objecttype)
         for item in itemlist.view:
             values = {}
-            values["ID"] = item
+            values["ID"] = str(item)
             if parents:
-                values["Parent"] = item.parent
+                values["Parent"] = str(item.parent)
 
             if itemtype != 'DataCenter' and summary:
-                values["summary"] = item.summary
```

Comment:
based on what pyvmomi returns it may be possible. the object is formated in some way which is kind of like json but isn't really. I hadn't updated that to convert to string as it still functions.
I can add a str conversion is that is prefered.